internal/store/static.go

```diff
@@ -114,20 +114,33 @@ func (store *StaticStore) GetPlugins(pluginFilter *model.PluginFilter) ([]*model
 }
 
 // getPlugins returns all plugins compatible with the given server version, sorted by name ascending.
-func (store *StaticStore) getPlugins(serverVersion string) ([]*model.Plugin, error) {
+func (store *StaticStore) getPlugins(serverVersion string, includeEnterprisePlugins bool) ([]*model.Plugin, error) {
 	var result []*model.Plugin
 	plugins := map[string]*model.Plugin{}
 
 	for _, storePlugin := range store.plugins {
-		if serverVersion != "" && storePlugin.Manifest.MinServerVersion != "" {
-			meetsMinServerVersion, err := storePlugin.Manifest.MeetMinServerVersion(serverVersion)
+		if serverVersion != "" {
```

Comment:
I thought about handling the case where `serverVersion == ""`. But the complexity didn't justified that value to me. The case `serverVersion == ""` should never happen with the mattermost server, right?

Reply:
In my opinion, the marketplace filters should work for non-mattermost-server clients. I often run a curl against the marketplace to check on the status, and I'd expect the `EnterprisePlugins` flag to work.

Reply:
Respecting the `curl` use case makes sense. I did updated the code.